data/implementations/discovery.json

```
@@ -19,6 +19,11 @@
           "name": "Go",
           "status": "Done",
           "url": "https://github.com/ipfs/go-ipfs/blob/master/core/bootstrap.go"
```

```suggestion
          "url": "https://github.com/ipfs/go-ipfs/blob/master/core/bootstrap/bootstrap.go"
```

Not familiar with the Golang codebase. Just a guess. The former is a dead link.